apps/cyberstorm-nextjs/.env.development

```diff
@@ -2,3 +2,10 @@
 # the security implications that entails.
 NEXT_PUBLIC_API_DOMAIN=http://thunderstore.localhost
 NEXT_PUBLIC_API_URL=$NEXT_PUBLIC_API_DOMAIN/api/cyberstorm/
+AUTH_ENABLED=
+AUTH_SECRET=development
+NEXT_PUBLIC_AUTH_BASE_URL=https://auth.thunderstore.io/auth/login
+NEXT_PUBLIC_AUTH_QP=?next=https%3A%2F%2Fbeta.thunderstore.io%2F
+NEXT_PUBLIC_AUTH_DISCORD_URL=$NEXT_PUBLIC_AUTH_BASE_URL/discord/
```

Comment:
Kinda odd to do this part of parsing in the env file but then do the return URL parsing in the component. Why not do both in the same place? Maybe here, since the URLs are defined once but possibly used in multiple places?

Not a blocker.

Reply:
My recollections of OAuth are a bit hazy but I'm not sure that would work. Doesn't the userneed to return to a login page that does some final validations? Any "redirect to original page" would have to be handled by that page? And the return URLs need to be registered with the OAuth provider anyway, and we don't want to register all the pages in the site. (Unless some wild card approach is possible.)